### ISSUE TITLE
Process:summarise: identify_tophit_and_contaminants2.py KeyError

### ISSUE THREAD

**Issue by WhalleyT** [Pathogen-Genomics-Cymru/lodestone#63]
KeyError when identify tophits can't find species key in dict:

```
  /bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8 LANG en_US.UTF-8 LANGUAGE en_US.UTF-8)
  Traceback (most recent call last):
    File "/opt/bin/identify_tophit_and_contaminants2.py", line 507, in <module>
      out, out_urls = process_reports(afanc_json, kraken_json, supposed_species, unmix_myco, myco_dir, prev_species_json, urls, tax_ids, sample_id)
    File "/opt/bin/identify_tophit_and_contaminants2.py", line 219, in process_reports
      for key in kraken['Species']:
  KeyError: 'Species'
```

for sample SRR26331619

**Comment by WhalleyT** [comment]
Looks like it's only finding non-TB/human hits on the Genus level

**Comment by WhalleyT** [comment]
Addressed in [79e0c8d](https://github.com/Pathogen-Genomics-Cymru/lodestone/commit/79e0c8d52abc784eab5d9e7eed944d848b8ca956). If there is no "Species" field found in the Kraken JSON, the script then tries for the Genus level, if it exists. If the Species is there then in breaks and only reports species. 

Needs to be approved by user group/ask their opinion.
